api/policies/hasValidMoveBody.js

```diff
@@ -89,6 +89,20 @@ module.exports = function (req, res, next) {
       }
       return next();
 
+    case MoveType.SEVEN_SCUTTLE:
```

Comment:
Because this is the exact same logic as the above SCUTTLE case, we should combine them so instead of adding logic here we just do 
```
  case MoveType.SCUTTLE:
  case MoveType.SEVEN_SCUTTLE:
```
Before the existing logic.

Reply:
While we are at it, let's also consolidate the `MoveType.JACK` case with these as well since they all have the same requirements